migrator/src/main/scala/akka/persistence/jdbc/migrator/package.scala

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>
+ * Copyright (C) 2019 - 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.jdbc
+
+import akka.persistence.jdbc.config.JournalConfig
+import akka.persistence.jdbc.journal.dao.JournalQueries
+import slick.jdbc.{
+  H2Profile,
+  JdbcBackend,
+  JdbcProfile,
+  MySQLProfile,
+  OracleProfile,
+  PostgresProfile,
+  SQLServerProfile
+}
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+package object migrator {
```

Comment:
I don't think we need to add a package object. We can just add a `DatabaseType.scala` and `JournalOrdering.scala` file. 

It makes it easier to find them when just browsing the code in GitHub, text editor or file browser. When we add traits and classes to a package.scala, we need to know they live there in order to find them.

Reply:
The ordering can never be respected because it is generated by the database. So the only way to ensure the ordering from the old journal to the new is to force insert them during the insertion into the new journal because the ordering field is a database auto-increment field. So after the migration of the data reset the ordering per database on the new journal as done here.

Reply:
but why wouldn't the ordering be respected if the records are inserted in the same order? True that the ordering is generated by the db, but it is deterministic, no? 

If we have a blank slate and we insert the first event, the ordering will be 1, second event ordering 2, etc. 

What am I missing?